lib/event_sourcery/memory/projector.rb

```diff
@@ -0,0 +1,25 @@
+module EventSourcery
+  module Memory
+    module Projector
+
+      def self.included(base)
+        base.include(EventSourcery::EventProcessing::EventStreamProcessor)
+        base.include(InstanceMethods)
+        base.class_eval do
+          alias_method :project, :process
+          class << self
+            alias_method :project, :process
+            alias_method :projects_events, :processes_events
+            alias_method :projector_name, :processor_name
+          end
+        end
```

Comment:
It's a shame we have to duplicate this (I assume it's the same code in the Postgres Projector). Could it live in EventStreamProcessor?